packages/pg-v5/lib/util.js

```diff
@@ -76,6 +75,7 @@ function presentCredentialAttachments (app, credAttachments, credentials, cred)
 exports.presentCredentialAttachments = presentCredentialAttachments
 
 exports.getConnectionDetails = function (attachment, config) {
+  const getBastion = require('./bastion').getBastion
```

Comment:
@uhoh-itsmaciek this more directly resolves the circular dependency. Do you think this is sufficient?

Reply:
Yeah, that looks reasonable. Thanks.